tests/browser/custom-reporter.ts

```diff
@@ -0,0 +1,88 @@
+import type {
+  FullConfig,
+  FullResult,
+  Reporter,
+  Suite,
+  TestCase,
+  TestResult,
+} from '@playwright/test/reporter';
+import { writeFileSync } from 'fs';
+import path from 'path';
+
+class MyReporter implements Reporter {
+  private passCount = 0;
+  private failCount = 0;
+  private skipCount = 0;
+  private failingTests: { path: string; name: string }[] = [];
+  private skippedTests: { path: string; name: string }[] = [];
+
+  onBegin(config: FullConfig, suite: Suite) {
```

Comment:
Nit: Is `config` required here as an argument since it is not used?

Reply:
@jack-ryan-nava-pbc, Looks like this parameter is required by the Reporter api, but I've renamed it to `_` to indicate that it's not used.